bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -89,10 +91,23 @@
 {% block content %}
 <main role="main">
   <header id="main-feature">
-      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <a href="{{ url('firefox') }}">{{ high_res_img('plugincheck/firefox-logo.png', {'alt': 'Mozilla Firefox', 'width': '145', 'class': 'firefox-logo'}) }}</a>
+      <h2 class="large">{{_('Check Your Plugins')}}</h2>
       <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
   </header>
 
+  <section class="newsletter-container">
+    <aside id="newsletter-subscribe">
+      <div class="content container">
+        {% if l10n_has_tag('plugincheck_newsletter') %}
+          {{ email_newsletter_form(title=_('<span class="firefox">Firefox</span> + You'), subtitle=_('Get Firefox tips, tricks, news and more'), button_class='white') }}
```

Comment:
We can surely do that, and it seems safe as strings have not been extracted or exposed.